articles/architecture-scenarios/implementation/b2b/b2b-authorization.md

```diff
@@ -22,14 +22,39 @@ useCase:
 
 <%= include('../../_includes/_authorization/_api-integration.md', { platform: 'b2b' }) %>
 
+## Role Based Access Control (RBAC)
+
+<%= include('../../_includes/_authorization/_rbac.md', { platform: 'b2b' }) %>
+
+The core RBAC feature can be used in many multi-organization scenarios.  See [Organization Data in an Access Tokens](#organization-data-in-an-access-token) for more information on how to ensure your setup can support your RBAC needs.
+
 ## Machine-to-Machine (M2M) authorization
 
 <%= include('../../_includes/_authorization/_m2m.md', { platform: 'b2b' }) %>
 
+## Organization Data in an Access Token
+
+If you have a separate API from your application in your system that supports your multi-organization application, it is important to restrict operations to only the organization that the token was generated for.  This requires that there is some sort of information in the access token to tell the API which organization the access token was issued for.  This can be done in a couple of different ways depending on the answers to a couple of simple questions:
```

Comment:
That's a good point, if you need consent, then you likely need to do a separate audience per organization as well because consent should be required for each organization.